### DISCUSSION TITLE
How to succesfully use code generation

### DISCUSSION THREAD

**Discussion by SakhileMamba** [rrousselGit/riverpod#2758] [Ideas]
I think the fact that developers have to add this:
part 'main.g.dart';

and run:
flutter pub run build_runner watch -d

to successfully use code generation should be clearly and boldly highlighted in the v2 docs.

**Comment by vicecaycedo** [comment]
Isn't that included in https://docs-v2.riverpod.dev/docs/getting_started?

**Comment by SakhileMamba** [comment]
Actually, it is. I think I'm the one that missed it.
